### PR TITLE
Use intent redirection to open konnector

### PR DESCRIPTION
**Pull request by CPatchane**
Hotfix to use `home` slug instead of `collect`

### CODE REVIEW

**Comment by CPatchane** [comment]
[[STO] Le bouton pour ouvrir un service/konnector est cassé](https://trello.com/c/qC15LiMB/1203-sto-le-bouton-pour-ouvrir-un-service-konnector-est-cass%C3%A9)

File: src/ducks/apps/index.js [hunk 1]
```diff
@@ -380,8 +380,8 @@ function onAppUpdate(appResponse) {
         'GET',
         `/${route}/${appResponse.slug}`
       )
-      // TODO throw error if collect is not installed
-      const collectApp = getState().apps.list.find(a => a.slug === 'collect')
+      // TODO Use intent REDIRECT instead
+      const collectApp = getState().apps.list.find(a => a.slug === 'home')
```
Comment (gregorylegarec) [inline src/ducks/apps/index.js]: s/collectApp/homeApp/

**Comment by gregorylegarec** [comment]
@CPatchane I updated the branch to use intent redirections, tell me what do you think about this.

File: src/ducks/apps/index.js [hunk 2]
```diff
@@ -312,19 +310,12 @@ export function fetchIconsProgressively() {
...
-  response,
-  collectLink,
-  fetchIcon = true
-) {
+export async function getFormattedInstalledApp(response, fetchIcon = true) {
   const appAttributes = _sanitizeManifest(response.attributes)
 
   let icon = response.links.icon
   if (fetchIcon) icon = await _getIcon(icon)
-  const openingLink =
```
Comment (CPatchane) [inline src/ducks/apps/index.js]: Is this variable `openingLink` still used here?

Comment (CPatchane) [reply src/ducks/apps/index.js]: Oh yeah you're right 👍 Sorry for that^^

**Pull request merged**